### ISSUE TITLE
Scribbles - Improving interactive segmentation algorithm (ETA 8/20)

### ISSUE THREAD

**Issue by masadcv** [Project-MONAI/MONAILabel#266]
This issue covers a number of minor bugs and improvements needed for the scribbles-based optimisation.

The current postproc sample app for scribbles implements the interactive segmentation part from https://arxiv.org/pdf/1710.04043.pdf 

There are a few discrepancies within the implementation of this method. These are:

1. Difference between the original equation and what has been implemented. In the paper, they set log prob to infinity which is equivalent of setting probability to 1 in the implementation. As we use probabilities, this needs to be fixed. 

2. The use of word "logits" may be misleading as we are using probabilities (post-softmax) instead of logits (pre-softmax). Either the logits volume need to be taken from pre-softmax or the term should be renamed to "prob".

3. The probability volumes from our CNN network are too "hard" meaning that they are mostly binary (either 0 or 1) in most places. This is proving to be problematic for the CRF optimisation, where the scribbles-based correction is not propagated to neighbouring visually "similar" voxels. Hence a method is needed that can soften these probabilities so that CRF is better aimed at correcting labels.

**Comment by masadcv** [comment]
This has been addressed in pull request https://github.com/Project-MONAI/MONAILabel/pull/272
